pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/containerd/logrotate/logrotate.go

```diff
@@ -0,0 +1,84 @@
+// Copyright (c) 2022 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logrotate
+
+import (
+	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
+	"k8s.io/utils/pointer"
+)
+
+// Config returns the content for logrotate units and files.
+// Whenever logrotate is ran, this config will:
+//   - keep only 14 old (rotated) logs, and will discard older logs.
```

Comment:
None of https://github.com/gardener/gardener/blob/f6cb9c29e13212676c4d84e57c5027b476b15c5d/pkg/operation/botanist/component/extensions/operatingsystemconfig/original/components/logrotate/logrotate.go#L26-L31 are applicable for containerd?

Reply:
nope